packages/testlab/package.json

```diff
@@ -4,10 +4,12 @@
   "description": "A collection of test utilities we use to write LoopBack tests.",
   "main": "index",
   "scripts": {
-    "build": "tsc && npm run build:es2015",
-    "build:es2015": "tsc --target es2015 --outDir lib-es2015",
+    "build": "npm run build:es6 && npm run build:es8",
+    "build:es6": "tsc --target es2015 --outDir lib6",
+    "build:es8": "tsc --target es2017 --outDir lib",
```

Comment:
Nitpick - feel free to ignore:

I don't think `es8` is a thing - we have es6 = es2015 and then they switched to year-based versioning -
 es2016, es2017, etc.

Maybe `build:lib` and `build:lib6` would make the intent more clear?